### ISSUE TITLE
Video player turning black after switching between apps

### ISSUE THREAD

**Issue by mustafah99** [lrorpilla/jidoujisho#352]
Video player turns black with only subtitles showing after switching between apps.

Found out about this issue after updating to OneUI 6 / latest Android release for Samsung Galaxy S23.

This wasn't an issue before the update. I used to minimize the video player and come back to the video being in the same state or just casually switch between apps from Jidoujisho to for example the web browser to search up certain sentences I found from the subs.

![Screenshot_20240207_161147](https://github.com/lrorpilla/jidoujisho/assets/49923247/fc483f5d-1993-48b4-97b6-02a791f1d218)

**Comment by Katsumi-III** [comment]
Same exact issue here. Coincidentally the same phone and software as you

**Comment by mustafah99** [comment]
> Same exact issue here. Coincidentally the same phone and software as you

Yeah, that's weird. It all happened after updating my system software.
